Tigran Kirakosov/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap" rel="stylesheet">
+  <title>Tokyo Weather Widget</title>
+</head>
+<body>
+  <article class="weather-container_main">
+    <div class="weather-container_widget">
+      <div class="weather-container_first-half">
+        <div class="current-weather__state"><img class="current-weather__state_image" src="./assets/images/icons/sun-ico.png" alt="sun icon"></div>
+        <p class="current-weather__desc">Sunny</p>
+        <p class="current-weather__date">Tokyo - Wednesday, July 31</p>
+      </div>
+      <ul class="weather-container_second-half">

```

Comment:
I think this ul is overhead - you have 3 absolutely different items inside (in general list items should be quite similar), but it's rather about semantic than some technical issue. more important - third element of list is `<ul>` - by specification children of `<ul>` should be just `<li>` elements (https://stackoverflow.com/questions/12242041/allowed-child-elements-of-ul)

Reply:
Yea, the first `<ul>` looks kinda weird and has no certain reason.
Well, I'll keep in mind the fact `<ul>` must contain nothing but `<li>` child elements.